libmscore/staff.h

```diff
@@ -279,6 +279,7 @@ class Staff final : public Element
     void setPlaybackVoice(int voice, bool val) { _playbackVoice[voice] = val; }
 
     std::array<bool, VOICES> visibilityVoices() const { return _visibilityVoices; }
+    bool voiceVisible(int voice) const { return _visibilityVoices[voice]; }
```

Comment:
I'd propose "isVoiceVisible" name

Reply:
also we need to gradually move implementation of methods into .cpp files

Reply:
fixed